torch_xla/distributed/spmd/xla_sharding.py

```diff
@@ -524,7 +530,7 @@ def mark_sharding(
 
         mesh (Mesh): describes the logical XLA device topology and the underlying device IDs.
 
-        partition_spec (Tuple[Tuple, int, str, None]): A tuple of device_mesh dimension index or
+        partition_spec (PartitionSpec): A tuple of device_mesh dimension index or
             `None`. Each index is an int, str if the mesh axis is named, or tuple of int or str.
             This specifies how each input rank is sharded (index to mesh_shape) or replicated (None).
             When a tuple is specified, the corresponding input tensor axis will be sharded along all
```

Comment:
This sentence is not complete. The corresponding input tensor axis will be sharded along all ... ???

Reply:
hmmm, it's pretty complete if i view the whole file